src/shared/main-styles.scss

```diff
@@ -1,47 +1,50 @@
-// $beer-text-color:black;
-// $bg-color:whitesmoke;
-// $navbar-text-color: white;
-// $title-color:black;
-// $navbar-bg-color:gray;
-// $button-positive-color:#4CAF50;
-// $button-previous-page-color:red;
-// $button-change-pictures-color:gray;
-// $button-text-color:white;
-// $input-border-color:green;
-// $input-text-color:black;
+$beer-text-color: black;
+$bg-color: whitesmoke;
+$navbar-text-color: white;
+$title-color: black;
+$navbar-bg-color: gray;
+$button-positive-color: #4caf50;
+$button-previous-page-color: red;
+$button-text-color: white;
+$button-positive-hover: #3e8e41;
+$input-border-color: green;
+$input-text-color: black;
+$delete-beer-button-color: red;
+$beer-text-size: 18px;
+$button-previous-page-color-hover: darkred;
+$description-name-color: maroon;
+$icon-favourite-bg-color: yellowgreen;
```

Comment:
Some colors repeat - they should not. This is caused by the naming convention that you follow.

Reply:
Always use color codes when describing colors. E.g. instead of `white` use `#ffffff` etc.

Reply:
Fixed